Sources/ComposableArchitecture/TestSupport/TestStore.swift

```diff
@@ -335,11 +344,19 @@
   }
 
   extension TestStore where LocalState: Equatable {
+    /// Sends an action to the store and asserts when state changes.
+    ///
+    /// - Parameters:
+    ///   - action: An action.
+    ///   - updateExpectingResult: A closure that asserts state changed by sending the action to the
+    ///     store. The mutable state sent to this closure must be modified to match the state of the
+    ///     store after processing the given action. Do not provide a closure if no change is
+    ///     expected.
     public func send(
       _ action: LocalAction,
+      _ updateExpectingResult: ((inout LocalState) throws -> Void)? = nil,
       file: StaticString = #file,
-      line: UInt = #line,
-      _ update: ((inout LocalState) throws -> Void)? = nil
+      line: UInt = #line
```

Comment:
Just wondering about this change, is there any specific reason you flip the order of this parameter @stephencelis ?

In my project, there is a need to provide the file and line.
Previous code looks like this:
```swift
store.send(.x, file: file, line: line) { ... }
```

but after this change, the trailing closure won't work and need to do this instead.
```swift
store.send(.x, { .... }, file: file, line: line)
```

Reply:
@jeffersonsetiawan Sorry about that! We didn't envision folks leveraging the `file:line:` defaults. The original order is an artifact of Swift not supporting trailing closure syntax in this case, but that was fixed a few versions back, so we decided to update things.